hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterStateManager.java

```diff
@@ -201,6 +203,10 @@ public boolean rollbackClusterState(String txnId) {
     }
 
     public void commitClusterState(ClusterState newState, Address initiator, String txnId) {
```

Comment:
Don't really need this method. It is only used from the tests and they can all just set the isTransient to false.